api/all.yaml

```diff
@@ -1490,6 +1811,202 @@ components:
         - READ_VOLUME
         - WRITE_VOLUME
       type: string
+    ModelInfo:
+      type: object
+      properties:
+        name:
+          description: Name of model, relative to parent schema.
+          type: string
+        catalog_name:
+          description: Name of parent catalog.
+          type: string
+        schema_name:
+          description: Name of parent schema relative to its parent catalog.
+          type: string
+        storage_location:
+          description: Storage root URL for model (for **MANAGED**, **EXTERNAL** tables)
```

Comment:
What does this mean? It seems to be relevant to UC tables, not models?

Reply:
Copy paste error.  Should read models